build/birt-packages/birt-report-all-in-one/BIRT.product

```diff
@@ -39,6 +39,8 @@ Powered by the Eclipse Platform: https://eclipse.org/eclipse
       </programArgs>
       <vmArgs>-Xms512m
 -Xmx2048m
+-Dorg.eclipse.swt.browser.DefaultType=edge
```

Comment:
I don't think we should set this for all operating system.  See `-Dorg.eclipse.swt.internal.carbon.smallFonts` below?